### ISSUE TITLE
Time series "points" member should be named "data" in QueryResult

### ISSUE THREAD

**Issue by prydin** [wavefrontHQ/python-client#21]
The query api doesn't work, since the returned object QueryResult and its embedded object Timeseries aren't mapping correctly to the returned json.

Consider running a GET-reguest like the following:

https://try.wavefront.com/api/v2/chart/api?q=ts%28%22vrops.cpu.demand%22,%20source=%22fde-vrava02%22%29&s=1505209901&g=m

The result will look something like this:

{ timeseries[ data: [ 1575783738, 7.9899 ] ... }

In other words, the data points are returned in the member named "data". The problem is that the member is named "points" in the QueryResults class, which causes the entire time series to be inaccessible through the Python API. To fix it, I suggest renaming the member in the Python object to "data".

**Comment by basilisk487** [comment]
Fixed in 2.2.1
